cmd/flags.go

```diff
@@ -43,8 +44,15 @@ func addonNameFromFlags(c *cli.Context, exitIfMissing ...bool) string {
 		os.Exit(1)
 	}
 
+	var addonUUID string
+	var err error
+	addonUUID, err = utils.GetAddonUUIDFromType(c.Context, app, addonName)
+	if err != nil {
+		fmt.Println("Unable to get the addon UUID based on its type:", err)
```

Comment:
From what I remember error message in the CLI must be displayed calling `io.Error` (https://github.com/Scalingo/cli/blob/7cf4f23d162908d11803e5c03d15f977d53059f7/io/status.go).